oozie-to-airflow/examples/shell/workflow.xml

```diff
@@ -0,0 +1,59 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<workflow-app xmlns="uri:oozie:workflow:1.0" name="shell-wf">
+    <start to="shell-node"/>
+    <action name="shell-node">
+        <shell xmlns="uri:oozie:shell-action:1.0">
+            <resource-manager>${resourceManager}</resource-manager>
+            <name-node>${nameNode}</name-node>
+            <prepare>
+                <create path="${nameNode}/examples/input-data/demo/pig-node"/>
+                <delete path="${nameNode}/examples/output-data/demo/pig-node"/>
+            </prepare>
+            <configuration>
+                <property>
+                    <name>mapred.job.queue.name</name>
+                    <value>${queueName}</value>
+                </property>
+            </configuration>
+            <exec>java</exec>
+            <argument>-version</argument>
+            <!--<file>${EXEC}</file>--> <!-- TODO Add the binary upload when file mapper ready -->
+            <capture-output/> <!-- This can be ignored by the converter -->
+        </shell>
+<!--        <ok to="check-output"/>-->
+        <ok to="end"/>
+        <error to="fail"/>
+    </action>
+<!--    TODO uncomment once decision has been implemented -->
```

Comment:
Remove it :) ?

Reply:
Yeah, I'll move it to a Github issue as a TODO.

Reply:
Done